core/ide/src/main/java/org/alice/stageide/sceneeditor/interact/GlobalDragAdapter.java

```diff
@@ -151,450 +141,152 @@ public boolean hasSceneEditor() {
   }
 
   private void setUpControls() {
```

Comment:
Good to see this simplified and cleaned up, but it (and `setupHandles` below) remain so long.
Is it worth breaking them up into smaller methods or other refactoring?

Reply:
yeah, I would like to keep working on breaking this up, especially setupHandles, but I was trying to stop shaving this yak 😅